fast-list.js

```diff
@@ -24,25 +29,27 @@ exports.scheduler = schedule;
 function FastList(source) {
   debug('initialize', source);
   this.editing = false;
+  this.source = source;
 
-  this.container = source.container;
-  this.container.style.overflowX = 'hidden';
-  this.container.style.overflowY = 'scroll';
-
-  this.list = source.list;
-  if (!this.list) {
-    this.list = document.createElement('ul');
-    this.container.appendChild(this.list);
+  // List elements aren't required
+  if (!source.list) {
+    source.list = document.createElement('ul');
+    source.container.appendChild(source.list);
```

Comment:
can be a follow up since it's not introduced in the PR, but we should maybe cover some of the "source fallbacks" with unit tests.